knowledge-base/chart-export-to-image.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Export Chart to image
+description: How to export the Chart to image
```

Comment:
The description must contain at least [100 characters](https://docs.telerik.com/style-guide/metadata).